modules/ROOT/pages/dataweave-cookbook-change-script-output-mime.adoc

```diff
@@ -0,0 +1,113 @@
+= Change a Script's MIME Type Output
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, transform, transformer, format, dataweave, data weave, output structure, mime type, mime
+
+This DataWeave example shows how to differentiate the output MIME type from the MIME type in which the output data is formatted. Such differentiation enables a Mule application to output a MIME type that clients can use to determine how to render the application's output data. An example is HTTP, which uses the MIME type to generate its `Content-Type` header.
+
+//LINK TO DW 1.0 EXAMPLES:
+include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
+
+The following example uses the "problem details" standard for APIs. The standard requires the
+`application/problem+json` content type in the response header and a JSON-formatted response body that contains specific keys.
+
+The example is a Mule flow that handles HTTP requests but always fails. The
+failure triggers the error handler, which sets the status code to 400 and uses
+a DataWeave script to produce a response that conforms to the standard.
+
+The DataWeave output directive, `output application/problem+json with json`, instructs
+the script to produce a response with the `application/problem+json` MIME type
+and a JSON-formatted body.
+
+A client that receives the 400 error uses the `Content-Type: application/problem+json` in the response to determine that it can handle the response data in the standard way, for example, using the `title` key it expects. By contrast, the MIME type `Content-Type: application/json`, without the `problem+`, fails to indicate whether the client can handle the error in that way.
```

Comment:
```suggestion
A client that receives the 400 error uses the `Content-Type: application/problem+json` in the response to determine that it can handle the response data in the standard way, for example, using the `title` key it expects. By contrast, the MIME type `Content-Type: application/json`, without `problem+`, fails to indicate whether the client can handle the error in that way.
```

Reply:
Hmm...I now see how long the example is.
Maybe break it into sections with the appropriate portion of the current intro text for each?